rotkehlchen/assets/spam_assets.py

```diff
@@ -757,95 +750,181 @@
-    and the list of spam assets KNOWN_ETH_SPAM_TOKENS. This function also makes sure to get the
-    bad assets in the list of cryptoscamdb and ensures that they exists in the globaldb before
-    trying to add them.
-
-    If `make_remote_query` is False then only the KNOWN_ETH_SPAM_TOKENS are added as
-    ignored assets.
-
-    TODO
-    This function tries to add as assets to the globaldb the tokens listed in
-    KNOWN_ETH_SPAM_TOKENS and not the ones coming from cryptoscamdb. The reason is that until the
-    v2 of the API the response contains both spam addresses and tokens and there is no way to know
-    if the address is for a contract or not. Checking if the address is a contract takes too much
-    time. When V2 gets released this can be fixed.
-    May raise:
-    - RemoteError
+def query_token_spam_list(db: 'DBHandler') -> set[EvmToken]:
+    """
+    Generate a set of assets that can be ignored using the list of spam
+    assets KNOWN_ETH_SPAM_TOKENS.
     """
     tokens_to_ignore = set()
-    if make_remote_query is True:
-        try:
-            response = requests.get(
-                url='https://api.cryptoscamdb.org/v1/addresses',
-                timeout=DEFAULT_TIMEOUT_TUPLE,
-            )
-            data = response.json()
-            success, tokens_info = data['success'], data['result']
-        except requests.exceptions.RequestException as e:
-            raise RemoteError(f'Failed to retrieve information from cryptoscamdb. {str(e)}') from e
-        except (DeserializationError, JSONDecodeError) as e:
-            raise RemoteError(f'Failed to deserialize data from cryptoscamdb. {str(e)}') from e
-        except KeyError as e:
-            raise RemoteError(
-                f'Response from cryptoscamdb doesn\'t contain expected key. {str(e)}',
-            ) from e
-
-        if success is False:
-            log.error(f'Failed to deserialize data from cryptoscamdb. {data}')
-            raise RemoteError(
-                'Failed to deserialize data from cryptoscamdb. Check the logs '
-                'to get more information',
-            )
-
-        for token_addr, token_data in tokens_info.items():
-            if not token_addr.startswith('0x') or token_data[0]['type'] != 'scam':
-                continue
-            try:
-                checksumed_address = to_checksum_address(token_addr)
-            except ValueError as e:
-                log.debug(f'Failed to read address from cryptoscamdb. {str(e)}')
-                continue
-            try:
-                token = EvmToken(ethaddress_to_identifier(checksumed_address))
-            except UnknownAsset:
-                continue
-            if token is not None:
-                tokens_to_ignore.add(token)
-
     # Try to add custom list
     for token_address, info in KNOWN_ETH_SPAM_TOKENS.items():
         try:
             own_token = get_or_create_evm_token(
```

Comment:
There is no inquirer passed, so there will be no remote calls, so there is no need to handle RemoteError and NotERC20Conformant. Or am I missing something?

Reply:
Yes, I believe there shouldn't be any query. The reason for it to be there is because this was not the previous behavior and I didn't update it. Done now